support-frontend/assets/components/paymentButton/defaultPaymentButtonContainer.tsx

```diff
@@ -4,6 +4,7 @@ import { currencies } from 'helpers/internationalisation/currency';
 import { getContributionType } from 'helpers/redux/checkout/product/selectors/productType';
 import { getUserSelectedAmount } from 'helpers/redux/checkout/product/selectors/selectedAmount';
 import { useContributionsSelector } from 'helpers/redux/storeHooks';
+import { shouldShowBenefitsMessaging } from 'pages/contributions-landing/components/DigiSubBenefits/helpers';
```

Comment:
Doesn't need to be done right now, but we should really move this out of the digi subs page folder.

Reply:
These were actually from some incorrectly stashed changes but totally agree!